woodpecker-server/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: woodpecker-server
 description: A Helm chart for the Woodpecker server
 type: application
-version: <version>
```

Comment:
As I understand `<version>` is replaced with real value by the release pipeline: https://github.com/woodpecker-ci/helm/blob/c319fca9ea79f90a3612561904f75c6f97c19168/.woodpecker.yml#L13 

So it will break the automated release process.

I'm guessing the main reason for this change is to enable installing from git. One way to enable it is to commit to manually updating the chart version every time when something changes. But it is quite a change in process and requires automation IMHO (e.g. run https://github.com/helm/chart-testing during CI to make sure version is updated on each chart change)

Reply:
Added the same comment in https://github.com/woodpecker-ci/helm/pull/13/files#r1133133744 since some discussion in going in this PR. Please lets use https://github.com/woodpecker-ci/helm/pull/13 to discuss the change.